### PR TITLE
Update to 3.0.1

### PR DESCRIPTION
**Pull request by rlaverde**
@goanpeca could you review it?

Should I add myself to recipe-mantainers?

### CODE REVIEW

**Comment by goanpeca** [comment]
@rlaverde (yes please add yourself)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ requirements:
     - flask
     - inifile
     - jinja2 >=2.4
-    - mistune
+    - mistune >=0.7.0,
```
Comment (goanpeca) [inline recipe/meta.yaml]: no coma needed (thats what happens when you copy from a setup.py directly :-p)

Comment (rlaverde) [inline recipe/meta.yaml]: 🙈

**Comment by rlaverde** [comment]
@goanpeca merge?

**Pull request merged**